<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="pathsWidget">
      <attribute name="title">
       <string>Paths</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="pathsLayout">
         <item row="0" column="1">
          <widget class="QLineEdit" name="mupen64Path"/>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="mupen64Button">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="dataButton">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="romPath"/>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="romButton">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="dataPathLabel">
           <property name="text">
            <string>Data directory:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="dataPath"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="pluginPathLabel">
           <property name="text">
            <string>Plugins directory:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pluginButton">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="mupen64PathLabel">
           <property name="text">
            <string>Mupen64Plus executable: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="romPathLabel">
           <property name="text">
            <string>ROM directory</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="pluginPath"/>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="configButton">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="configPath"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="configPathLabel">
           <property name="text">
            <string>Config directory:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="pathsSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="emulationWidget">
      <attribute name="title">
       <string>Emulation</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QGridLayout" name="emulationLayout">
         <property name="topMargin">
          <number>10</number>
         </property>
         <item row="1" column="0">
          <widget class="QRadioButton" name="cachedButton">
           <property name="text">
            <string>Cached Interpreter</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="pureButton">
           <property name="text">
            <string>Pure Interpreter</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="dynamicButton">
           <property name="text">
            <string>Dynamic Recompiler</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="graphicsWidget">
      <attribute name="title">
       <string>Graphics</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <layout class="QGridLayout" name="graphicsLayout" columnminimumwidth="150,0">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="osdLabel">
           <property name="text">
            <string>On Screen Display:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="fullscreenLabel">
           <property name="text">
            <string>Fullscreen:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="resolutionLabel">
           <property name="text">
            <string>Resolution:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="osdOption">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="fullscreenOption">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="resolutionBox">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="graphicsVSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="graphicsHSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pluginsWidget">
      <attribute name="title">
       <string>Plugins</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0">
        <layout class="QGridLayout" name="pluginsLayout" columnstretch="0,1" columnminimumwidth="100,0">
         <item row="0" column="0">
          <widget class="QLabel" name="videoLabel">
           <property name="text">
            <string>Video Plugin:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="audioLabel">
           <property name="text">
            <string>Audio Plugin:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="inputLabel">
           <property name="text">
            <string>Input Plugin:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="rspLabel">
           <property name="text">
            <string>RSP Plugin:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="videoBox"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="audioBox"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="inputBox"/>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="rspBox"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="pluginsVSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="columnsWidget">
      <attribute name="title">
       <string>Columns</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="2">
        <spacer name="columnsHSpacer2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="columnsLayout">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QListWidget" name="availableList">
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>250</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QWidget" name="sortWidget" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="sortLayout" stretch="0,0">
              <item>
               <widget class="QToolButton" name="sortDownButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="arrowType">
                 <enum>Qt::UpArrow</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="sortUpButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="arrowType">
                 <enum>Qt::DownArrow</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QListWidget" name="currentList">
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>250</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="currentLabel">
           <property name="text">
            <string>Current:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="availableLabel">
           <property name="text">
            <string>Available:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="stretchOption">
           <property name="text">
            <string>Stretch first column</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QWidget" name="toggleWidget" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="toggleLayout">
              <item>
               <widget class="QToolButton" name="addButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="arrowType">
                 <enum>Qt::RightArrow</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="removeButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="arrowType">
                 <enum>Qt::LeftArrow</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <spacer name="columnsVSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <spacer name="columnsHSpacer1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="otherWidget">
      <attribute name="title">
       <string>Other</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="otherLayout" columnminimumwidth="100,0">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="saveLabel">
           <property name="text">
            <string>Save Options:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="saveOption">
           <property name="toolTip">
            <string extracomment="If checked, CEN64's output will be redirected to CEN64-Qt so you can view the VI/s or debug information."/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="otherVSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="otherHSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>mupen64Path</tabstop>
  <tabstop>romPath</tabstop>
  <tabstop>pluginPath</tabstop>
  <tabstop>dataPath</tabstop>
  <tabstop>configPath</tabstop>
  <tabstop>mupen64Button</tabstop>
  <tabstop>romButton</tabstop>
  <tabstop>pluginButton</tabstop>
  <tabstop>dataButton</tabstop>
  <tabstop>configButton</tabstop>
  <tabstop>pureButton</tabstop>
  <tabstop>cachedButton</tabstop>
  <tabstop>dynamicButton</tabstop>
  <tabstop>osdOption</tabstop>
  <tabstop>fullscreenOption</tabstop>
  <tabstop>resolutionBox</tabstop>
  <tabstop>videoBox</tabstop>
  <tabstop>audioBox</tabstop>
  <tabstop>inputBox</tabstop>
  <tabstop>rspBox</tabstop>
  <tabstop>availableList</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>currentList</tabstop>
  <tabstop>sortDownButton</tabstop>
  <tabstop>sortUpButton</tabstop>
  <tabstop>stretchOption</tabstop>
  <tabstop>saveOption</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
